pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -35,10 +34,10 @@ const (
 )
 
 var (
-	knativeVersion = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "knative_serving_version",
-		Help: "Installed knative serving version info",
-	}, []string{"version"})
+	servingHealth = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "knative_serving_health",
+		Help: "health status of knative serving",
+	}, []string{"healthState", "dependenciesInstalled", "deploymentsAvailable", "installSucceeded"})
```

Comment:
Do we need "healthState"? That's already expressed via 0 or 1 on the metric itself I feel?